lib/board.rb

```diff
@@ -0,0 +1,81 @@
+class Board
```

Comment:
SRP: The details of these responsibilities are mixed together: modelling the cells of the grid, modelling the lines, modelling the rules of the game.